### PR TITLE
added blog post for new site launch

### PR DESCRIPTION
**Pull request by yochannah**

### CODE REVIEW

File: _posts/2018-09-16-launching-new-site.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+We've had two fantastic [Google Summer of Code](https://summerofcode.withgoogle.com/) students, Sarthak Sehgal and Megh Thakkar, working with us for the last few months. Over this time they've completely redesigned both [the look](https://github.com/biojs/biojs-frontend) (implemented in Vue) and [backend](https://github.com/biojs/biojs-backend) of the BioJS component registry. We're extremely proud to announce that the new site will be launching today (16 September 2018).
+
+We'll also be migrating our domains slightly to reduce maintenance burden. Please be patient if you notice glitches over the next couple of days while DNS records update.
+
+## Here's what will change:
+
+- [biojs.net](http://www.biojs.net) and [biojs.io](http://www.biojs.io) will both point to the new registry. There will no longer be a homepage separate to the registry.
+- The BioJS blog will live at [blog.biojs.net](http://www.biojs.net). This is [hosted on GitHub](https://github.com/biojs/biojs.github.io) using GitHub pages, which should allow anyone to make PRs to add blog posts if they wish (Do you use or create BioJS components? We'd love to host a guest post of yours!)
```
Comment (DennisSchwartz) [inline _posts/2018-09-16-launching-new-site.md]: The blog url is wrong in the link display :)
If you don't have time I can fix it as well.

**Comment by yochannah** [comment]
@DennisSchwartz Thanks for the review and good spot! I've fixed it now, can you double-check and approve if it looks better?

**Comment by DennisSchwartz** [comment]
@yochannah great! Sorry this took so long haha.

**Comment by yochannah** [comment]
Thanks Dennis!

**Pull request merged**